node/sc/sub_bridge_handler.go

```diff
@@ -383,26 +390,18 @@ func (sbh *SubBridgeHandler) handleParentChainInvalidTxResponseMsg(msg p2p.Msg)
 	if err := msg.Decode(&invalidTxs); err != nil && err != rlp.EOL {
 		return errResp(ErrDecode, "msg %v: %v", msg, err)
 	}
-	txPool := sbh.subbridge.GetBridgeTxPool()
 	for _, invalidTx := range invalidTxs {
```

Comment:
```suggestion
        txPool := sbh.subbridge.GetBridgeTxPool()
	for _, invalidTx := range invalidTxs {
```
I think we don't have to get txPool newly when we loop everytime.

Reply:
Thanks. changed.